runtime/src/lib.rs

```diff
@@ -102,8 +121,8 @@ pub mod opaque {
 
 /// This runtime version.
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-    spec_name: create_runtime_str!("academy-pow"),
-    impl_name: create_runtime_str!("academy-pow"),
+    spec_name: create_runtime_str!("frontier-template"),
+    impl_name: create_runtime_str!("frontier-template"),
```

Comment:
Do we want this?

Reply:
This is actually an important hack. This is what tells Polkadot JS and the tools that build on it (like the contracts-ui) to use ethereum-style accounts. I guess it warrants a comment actually.

```suggestion
    // We use this name as a hint to Polkadot JS to use Ethereum signatures.
    // see https://github.com/polkadot-js/apps/issues/9632 for the status on a proper fix.
    spec_name: create_runtime_str!("frontier-template"),
    impl_name: create_runtime_str!("frontier-template"),
```